tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -10962,6 +10962,20 @@ def test_tag_that_looks_like_aggregate(self):
         assert len(data) == 1
         assert data[0]["p95"] == "<5k"
 
+    def test_chained_or_query_meta_tip(self):
+        query = {
+            "field": ["transaction"],
+            "query": "transaction:a OR transaction:b",
+            "project": [self.project.id],
+        }
+        response = self.do_request(query)
+        assert response.status_code == 200, response.content
+        meta = response.data["meta"]
+        assert meta["tips"] == {
+            "query": "Did you know you can replace chained or conditions like `field:a OR field:b OR field:c` with `field:[a,b,c]`",
```

Comment:
wouldn't it be better if we handle the phrases on the frontend using one of our translation functions?

Reply:
Do we do anything like that for our 400 errors?